RadixWallet/Features/SettingsFeature/ImportFromOlympiaLegacyWallet/Coordinator/ImportOlympiaWalletCoordinator+View.swift

```diff
@@ -51,9 +51,7 @@ extension ImportOlympiaWalletCoordinator.Path {
 						/State.importMnemonic,
 						action: Action.importMnemonic,
 						then: { childStore in
-							NavigationView {
-								ImportMnemonic.View(store: childStore)
-							}
+							ImportMnemonic.View(store: childStore)
```

Comment:
@kugel3 reverting the change you made recently, which broke toolbar in Import from Olympia, which combined with the 3 letter bug, prevents users from inputting some mnemonics (containing 3 letter words which are ambigious with 3 letters)

Do you remember why you made the change? relating to navigation title perhaps?

Reply:
Yes, there should be a navigation view at the top level, it looks very strange without it. But can't we have it _only_ there, and ditch the dummy one?

Reply:
ImportMnemonic is used from many places in the app:
* Olympia migration
* Restore wallet - part of ImportManyMnemonicsFlow
* Import from Account (should be merge with bottom)
* Import from Display Seed Phrases

So having it in ImportMnemonic itself ensure toolbar is always visible disregarding of starting point. But I guess we have no option but to manually ensure it is visible on all places...? Bah.... ought to be an easier way of having a toolbar :/ 

Anyway I will take a look

Reply:
fixing toolbar was tricker than I had hoped, so will be done in a separate PR,